modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/maven/Gpg.kt

```diff
@@ -42,9 +44,30 @@ class Gpg {
                 ascFile.delete()
                 val allArgs = arrayListOf<String>()
                 allArgs.add(gpg)
+
+                val pwd = localProperties.getNoThrows("gpg.password")
+                if (! pwd.isNullOrBlank()) {
```

Comment:
I'm surprised you don't need `?.` here, `getNoThrows()` returns a `String?`, right?